site/components/Navigation/index.tsx

```diff
@@ -58,47 +63,283 @@ export const Navigation: FC<Props> = ({
+          buttons={[
+            <ButtonPrimary
+              label="INTRODUCTION"
+              key="intro"
+              href="/blog"
+              className={styles.navMain_DesktopMenuItem}
+            />,
+            <ButtonPrimary
+              label="HOT TO PLEDGE"
+              key="how pleadge"
+              href="/blog"
+              className={styles.navMain_DesktopMenuItem}
+            />,
+            <ButtonPrimary
+              label="LEADER BOARD"
+              key="leader board"
+              href="/blog"
+              className={styles.navMain_DesktopMenuItem}
+            />,
+          ]}
         />
         <NavItemDesktop
           url={urls.loveletter}
           name={t({ message: "Love Letters", id: "shared.loveletters" })}
         />
         <NavItemDesktop
-          url="/blog"
           name={t({ message: "Resources", id: "shared.resources" })}
           active={activePage === "Resources"}
+          buttons={[
+            <ButtonPrimary
+              label="HOW TO BUY KLIMA"
+              key="how to buy"
+              href="/blog"
+              className={styles.navMain_DesktopMenuItem}
+            />,
+            <ButtonPrimary
+              label="CARBON DASHBOARDS"
+              key="carbon dashboards"
+              href="/blog"
+              className={styles.navMain_DesktopMenuItem}
+            />,
+            <ButtonPrimary
+              label="DISCLAIMER"
+              key="disclaimer"
+              href="/blog"
+              className={styles.navMain_DesktopMenuItem}
+            />,
+            <ButtonPrimary
+              label="DOCS"
+              key="docs"
+              href="/blog"
+              className={styles.navMain_DesktopMenuItem}
+            />,
+          ]}
         />
       </HeaderDesktop>
 
       <HeaderMobile
-        buttons={[
-          <ChangeLanguageButton key="ChangeLanguageButton" />,
-          ...(showThemeToggle ? [<ThemeToggle key="ThemeToggle" />] : []),
-        ]}
+        buttons={[<ChangeLanguageButton key="ChangeLanguageButton" />]}
       >
-        <NavItemMobile
-          url={createLinkWithLocaleQuery(urls.offset, locale)}
-          name={t({ message: "Offset", id: "shared.offset" })}
-        />
-        <NavItemMobile
-          url="/infinity"
-          name={t({ message: "Infinity", id: "shared.infinity" })}
-        />
-        <NavItemMobile
-          url={urls.loveletter}
-          name={t({ message: "Love Letters", id: "shared.loveletters" })}
-        />
-        <NavItemMobile
-          url="/blog"
-          name={t({ message: "Resources", id: "shared.resources" })}
-        />
+        <div className={styles.navMain_MobileItemsWrapper}>
+          <div className="links">
+            <NavItemMobile
+              url={createLinkWithLocaleQuery(urls.offset, locale)}
+              name={t({ message: "About", id: "shared.about" })}
+              selected={selected}
+              setSelected={setSelected}
```

Comment:
As you are passing hard coded strings around to toggle the NavItemMobile, I would make that more clear and rename:
- `selected` -> `toggledNavItemID `
- `setSelected` -> `setToggledNavItemID `

and as mentioned above move this state handling to `NavMobile`

Reply:
updated